saleor/graphql/account/mutations/account.py

```diff
@@ -36,18 +43,48 @@ class Arguments:
             description="Fields required to create a user.", required=True
         )
 
+    requires_confirmation = graphene.Boolean(
+        description="Informs whether users need to confirm their email address."
```

Comment:
Let's add `required=True` since this field will always return some boolean value and never a `null`.